src/app/pages/TopicPage/TopicPage.jsx

```diff
@@ -31,6 +36,12 @@ const TopicPage = ({ pageData }) => {
   const { lang, translations } = useContext(ServiceContext);
   const { title, description, promos, pageCount, activePage } = pageData;
 
+  const { enabled: adsEnabled } = useToggle('ads');
+  const { isAmp, showAdsBasedOnLocation } = useContext(RequestContext);
```

Comment:
We don't support amp for topic pages afaik, could we remove the amp checking logic?